euth_wagtail/assets/scss/components/_dashboard.scss

```diff
@@ -174,9 +182,37 @@ h1.dashboard-title {
     }
 
     .blueprint-phases {
-        &-item {
-            max-width: 30%;
-            vertical-align: top;
+        ul {
+            display: inline-block;
+        }
+
+        li {
+            overflow: hidden;
+            @include rem(padding-bottom, 10px);
+            @include rem(width, 300px);
+        }
+
+        .icon {
+            float: left;
+            @include rem(padding-bottom, 4px);
+            @include rem(padding-right, 10px);
+        }
+
+        .description {
+            text-align: left;
+        }
+
+        .circled {
```

Comment:
didn't know the stacking feature. nice. fixed in 4b88751726b11b317e429c920475f6df9a6d8def